testing/librtas/APKBUILD

```
@@ -0,0 +1,35 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=librtas
+pkgver=2.0.1
+pkgrel=0
+pkgdesc="Librtas provides a set of libraries that access Run-Time Abstraction Services (RTAS)"
+url="https://github.com/nfont/librtas"
+arch="all"
+license="LGPL"
+depends=""
+makedepends="autoconf automake libtool"
+install=
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/nfont/librtas/archive/v$pkgver.tar.gz"
+
+_builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$_builddir"
+	./autogen.sh
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$_builddir"
+	make DESTDIR="$pkgdir" install
+	return 0
```

This is wrong, why you want the package to silently pass failed `make install`?